tests/selectors.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { expect, test } from "@playwright/test";
+
+import { AllPages } from "../pages/allpages";
+import { TIMEOUT } from "dns";
+import { setTimeout } from "timers";
+
+test.describe("Selectors", () => {
+    let pages: AllPages;
+
+    test.beforeEach(async ({ page }) => {
+        pages = new AllPages(page);
+        await pages.selectorsPage.visit();
+    });
+
+    /*test("click on fox picture", async ({ page }) => {
+    await pages.findFoxPage.foxpicturebutton.
+    click({timeout:30000});
+    await expect(pages.findFoxPage.textBelowImage).toHaveText("Gratuluji! Tvůj úkol je splněn! Vychutnej si ohňostroj s 🦊")
+    });
+  */
+
+    test("find data-qa element", async ({ page }) => {
+        await pages.selectorsPage.elementByData;
+        await expect(pages.selectorsPage.elementByData).toBeVisible
+    });
+
+    test("find id zapletka element", async ({ page }) => {
+        await pages.selectorsPage.elementIdZapletka;
+        await expect(pages.selectorsPage.elementIdZapletka).toBeVisible
+    });
+
+    test("find elements with this-is-interesting-paragraph class", async ({ page }) => {
+        await pages.selectorsPage.elementInteresting;
+        await expect(pages.selectorsPage.elementInteresting).toBeVisible
+    });
```

Comment:
`toBeVisible` je funkce - takze tady potrebujes mit `toBeVisible()` - pak to zacne vyhazovat chybicky tohoto typu:

```
Error: expect.toBeVisible: Error: strict mode violation: locator('.this-is-interesting-paragraph') resolved to 2 elements:
    1) <p class="this-is-interesting-paragraph">↵          Liška hrášek snědla, vzala Budulínka n…</p> aka getByText('Liška hrášek snědla, vzala')
    2) <p class="this-is-interesting-paragraph">↵          Dědeček s babičkou přišli domů a hleda…</p> aka getByText('Dědeček s babičkou přišli dom')


Call log:
  - expect.toBeVisible with timeout 5000ms
  - waiting for locator('.this-is-interesting-paragraph')

    at tests/selectors.spec.ts:34:62
```

Mrknes na to?